packages/webapp/pages/account/notifications.tsx

```diff
@@ -166,8 +175,6 @@ const AccountNotificationsPage = (): ReactElement => {
       NotificationCategory.Product,
     );
 
-    onToggleReadingReminder(!isSubscribed);
-
```

Comment:
Removed it from onTogglePush, so it doesn't get automagically enabled on notification toggle.

Reply:
Yeah somehow totally missed this one 👍